lib/kitchen/directions/bake_custom_sections/v1.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module Kitchen::Directions::BakeCustomSections
+  class V1
+    def bake(chapter:)
+      custom_sections_properties = {
+        narrative_trailblazer: {
+          class: 'narrative-trailblazer',
+          text: 'Literacy Narrative Trailblazer',
+          inject: 'title'
+        },
+        living_words: {
+          class: 'living-words',
+          text: 'Living by Their Own Words',
+          inject: 'subtitle'
+        },
+        quick_launch: {
+          class: 'quick-launch',
+          text: 'Quick Launch: ',
+          inject: 'title_prefix'
+        },
+        drafting: {
+          class: 'drafting',
+          text: 'Drafting: ',
+          inject: 'title_prefix'
+        },
+        peer_review: {
+          class: 'peer-review',
+          text: 'Peer Review: ',
+          inject: 'title_prefix'
+        },
+        revising: {
+          class: 'revising',
+          text: 'Revising: ',
+          inject: 'title_prefix'
+        }
+      }
+
+      custom_sections_properties.each do |_custom_section_name, property|
+        property_classes = property[:class]
+        inject = property[:inject]
+
+        chapter.search(".#{property_classes}").each do |custom_section|
```

Comment:
@LostMojo About make searching more specific, it can't be done in this case, because some of the classes concern sections but one of them is present in page tag. It is visible in the spec file: 
![image](https://user-images.githubusercontent.com/55203598/129046365-9ef4d62e-7df7-4924-a44c-a4393551dcf7.png)
The only option here is splitting the iteration into two loops one for searching the sections and another for pages, but this could be problematic for implementing the `case` statement which makes this whole module code clearer. I also don't know if it is worth the effort, because two loops will  make the time of baking longer too. Let me know what are your thoughts. :)